src/apis/groo/config.ts

```diff
@@ -69,7 +69,8 @@ axiosGroo.interceptors.response.use(
           // 토큰 재발행이 아닌 요청에서 401을 받은 경우
           try {
             // 토큰 재발급 API 호출
-            await fetchGroo.auth.reissueToken();
+            // fetchGroo.auth.reissueToken() 대신 직접 호출
+            await axiosGroo.post('/token-refresh');
```

Comment:
```suggestion
            await axiosGroo.post('/token-refresh');
```
에러 코드는 주석으로 남겨두지말고 지워도 될 것 같습니다!